cxplat/src/cxplat_winuser/cxplat_winuser.cpp

```diff
@@ -6,6 +6,7 @@
 #include "leak_detector.h"
 #include "symbol_decoder.h"
 
+#include <Psapi.h>
```

Comment:
does this need to go before the other includes or can it appear in alphabetical order?